tests/unit/gapic/logging_v2/test_logging_service_v2.py

```diff
@@ -71,6 +71,23 @@ def modify_default_endpoint(client):
     )
 
 
+def test_logging_default_client_info_headers():
+    import re
```

Comment:
Seems like repeated code - can you create one function controlled/operated by parameters?

Reply:
Yeah unfortunately a lot of these test files contain mostly repeated test code pointed at different services. I considered making a different file for just these tests that could share more logic, but then decided to follow the existing conventions instead. Let me know if you think that makes sense